app.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+function StoreSalesGenerator(name, minCust, maxCust, avgSale) {
+  this.storeName = name;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgSale = avgSale;
+  this.numCookies = [];
+  this.totalSales = 0;
+  this.hour = 6;
+}
+
+var newText = document.createTextNode('text');
+var tableRow, tableHeader, tableData;
+var position = document.getElementById('sales-estimates');
+
+var storeHour = function(){
+  tableRow = document.createElement ('tr');
+  position.appendChild(tableRow);
+  tableData = document.createElement ('td');
+  tableRow.appendChild(tableData);
+  tableData = document.createElement ('td');
+  for (var hour = 6; hour < 20; hour++ ) {
+    tableHeader = document.createElement ('th');
+    if (hour < 12 ) {
+      newText = document.createTextNode(hour + ' am ');
+      tableHeader.appendChild(newText);
+      tableRow.appendChild(tableHeader);
```

Comment:
As these two lines are repeated in every block of this if/else if/else set you should move those following the else.